invoiceregistry_test.go

```diff
@@ -320,6 +320,39 @@ func TestNoSubscriberFail(t *testing.T) {
 	c.checkHtlcFailed(ResultNoAcceptSubscriber)
 }
 
+// TestGracePeriodForSubscriber asserts that a new set is started even if the last subscriber has recently disconnected.
+// After the grace period, we should not accept any new sets
+func TestGracePeriodForSubscriber(t *testing.T) {
```

Comment:
There was already a test for no subscribers, `TestNoSubscriberFail`. I think this may overlap partially with this test?

Reply:
First I thought, it should be 1 test, 1 feature

TestNoSubscriberFail --> test when no subscribers were connected

TestGracePeriodForSubscriber --> test graceful period

But you are right, we can consider this as the same feature maybe

I will merge it